### PR TITLE
[4.0] Tidy up spacing on plugin descriptions

### PR DESCRIPTION
**Pull request by particthistle**
Pull Request for Issue #31267 

### Summary of Changes
- ~~Changes `<br>` to `<br />` consistently throughout plugin language files.~~ 
- ~~Changes `<br>` to `<br><br>` in various language files to add an additional line before warnings in descriptions to improve readability on screen.~~
- Changes `<br>` to paragraphs in various language files to add spacing and formatting before warnings in descriptions to improve readability on screen.

Where @gostn identified Content - Fields plugin having the issue in #31267 a closer inspection of plugins showed a number of plugins with descriptions that could be improved by additional spacing being added.

### Testing Instructions
- Before applying the patch, check a few of the plugins in the list to see what the before status looks like.
- Apply the patch
- Check the various plugins listed in the file changes for this PR to see the various changes have taken effect without impacting the usability of the plugin details.

### Actual result BEFORE applying this Pull Request
Before spacing is improved, there will not be a space between the first paragraph and the second paragraph:
image

### Expected result AFTER applying this Pull Request
After spacing is improved, there will be an additional space between the first paragraph and the second paragraph:
image

### Documentation Changes Required
None

### CODE REVIEW

**Comment by richard67** [comment]
Sure we need the `<br>` to `<br />` change? As far as I know, HTML5 is the default for J4, and there a stand-alone `<br>` is absolutely valid, and the `<br />` is the J3 XHTML style which we want to get rid of in J4. But I might be wrong, so hoping for more feedback.

**Comment by brianteeman** [comment]
Sorry but the change to `<br />` is not correct. They were all changed to `<br>` deliberately

**Comment by brianteeman** [comment]
If you believe that the spacing between the sentences should be greater (i have no opinion on this) then it must be done as paragraphs and not with double `<br>` 

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br for an explanation

**Comment by gostn** [comment]
I have tested this item :white_check_mark: successfully on 307e2029e4ca13b25dae3f843887505bc8086bfd<br /><br />Thanks @particthistle <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by richard67** [comment]
> Sorry but the change to `<br />` is not correct. They were all changed to `<br>` deliberately

> If you believe that the spacing between the sentences should be greater (i have no opinion on this) then it must be done as paragraphs and not with double `<br>`
> 
> See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br for an explanation

@brianteeman Thanks for your feedback. It's consistent with the information I have.

@particthistle Can you modify this PR to fix the issue in the right way?

Setting the "Updates Requested" label.

**Comment by adj9** [comment]
I have tested this item :white_check_mark: successfully on 307e2029e4ca13b25dae3f843887505bc8086bfd<br /><br />done :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by richard67** [comment]
@gostn @adj9 When you are testing, do you completely ignore that there are review comments with valid requests for changes?

**Comment by gostn** [comment]
@richard67 you completely ignore the possibility that one test while other write requests.

**Comment by particthistle** [comment]
Apologies - I think I'd looked at a J3 language file and had seen the `<br />`

Reverted the changes back to use <br> only. Number of files edited now has dropped from 26 to 17 as several had only been the modification to `<br />`

**Comment by richard67** [comment]
> If you believe that the spacing between the sentences should be greater (i have no opinion on this) then it must be done as paragraphs and not with double `<br>`
> 
> See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br for an explanation

This still needs to be addressed.

Question: Couldn't the spacing issue be solved with CSS?

**Comment by jiweigert** [comment]
I would go with a paragraph ```<p>``` as  Brian Teeman suggested, this would show a spacing in any case, even the CSS is not loaded.
If its not fitting the styleguides, there is still the option to adjust it via CSS later.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by adj9** [comment]
@richard67 
I have not considered this feature!!! I have not considered it!I have not considered this feature!!! I have not considered it!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by ceford** [comment]
There are 550 instances of &lt;br /&gt; in the code. And 1536 instances of &lt;br&gt;. Why not do them all?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by brianteeman** [comment]
Dont know where you are looking but this PR is for J4 and there should be zero instances of `<br />`

**Comment by ceford** [comment]
On the number of instances of &lt;br /&gt; - I searched my fork of 4.0-dev branch of the joomla-cms repo. I did not check where the instances occur: mostly in libraries/vendor, media/vendor, and node/modules. There is one occurrence in administrator/components/com_installer/tmpl/updatesite/default.php and another in configuration.php (off-line message). So I admit to making a mountain out of a molehill! Sorry.

**Comment by brianteeman** [comment]
That's not a mountain its the entire Himalaya ;)

**Comment by particthistle** [comment]
Let's try that again. Misinterpreted earlier comments and so had just corrected the `<br>` and not converted to `<p>`.

Now converted to paragraphs where it then improves the flow of the descriptions of the various plugins.

Also pushed a commit to correct the `<br />` to `<br>` in the file @ceford found in:
administrator\components\com_installer\updatesites\default.php

**Comment by gostn** [comment]
@richard67 please add label ´Ready to Test´ to pr´s so https://github.com/joomla/joomla-cms/pull/31285#issuecomment-719955070 can never happen again.

File: administrator/components/com_installer/tmpl/updatesites/default.php [hunk 1]
```diff
@@ -107,7 +107,7 @@
 									<span class="small break-word">
 										<a href="<?php echo $item->location; ?>" target="_blank" rel="noopener noreferrer"><?php echo $this->escape($item->location); ?></a>
 									</span>
-									<br />
+									<br>
```
Comment (brianteeman) [inline administrator/components/com_installer/tmpl/updatesites/default.php]: Your branch must be out of date as this has already been fixed #31301

Comment (particthistle) [reply administrator/components/com_installer/tmpl/updatesites/default.php]: It was.

File: administrator/language/en-GB/plg_content_pagebreak.ini [hunk 2]
```diff
@@ -17,4 +17,4 @@ PLG_CONTENT_PAGEBREAK_SLIDERS="Sliders"
...
+PLG_CONTENT_PAGEBREAK_XML_DESCRIPTION="<p>Allow the creation of a paginated article with an optional table of contents.</p><p>Insert page breaks through the use of the page break button normally found in the WYSIWYG editor toolbar. The location of the page break in an article will be displayed in the editor as a simple horizontal line.</p><p>The text displayed will depend on the options chosen and may be either the title, alternate text (if provided) or page numbers. </p><p>The HTML usage is:<br>&lt;hr class=&quot;system-pagebreak&quot; /&gt;<br>&lt;hr class=&quot;system-pagebreak&quot; title=&quot;The page title&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; alt=&quot;The first page&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; title=&quot;The page title&quot; alt=&quot;The first page&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; alt=&quot;The first page&quot; title=&quot;The page title&quot; /&gt;</p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_content_pagebreak.ini]:
```suggestion
PLG_CONTENT_PAGEBREAK_XML_DESCRIPTION="<p>Allow the creation of a paginated article with an optional table of contents.</p><p>Insert page breaks through the use of the page break button normally found in the WYSIWYG editor toolbar. The location of the page break in an article will be displayed in the editor as a simple horizontal line.</p><p>The text displayed will depend on the options chosen and may be either the title, alternate text (if provided) or page numbers.</p><p>The HTML usage is:<br>&lt;hr class=&quot;system-pagebreak&quot; /&gt;<br>&lt;hr class=&quot;system-pagebreak&quot; title=&quot;The page title&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; alt=&quot;The first page&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; title=&quot;The page title&quot; alt=&quot;The first page&quot; /&gt; or <br>&lt;hr class=&quot;system-pagebreak&quot; alt=&quot;The first page&quot; title=&quot;The page title&quot; /&gt;</p>"
```

File: administrator/language/en-GB/plg_authentication_cookie.ini [hunk 3]
```diff
@@ -8,4 +8,4 @@ PLG_AUTHENTICATION_COOKIE_ERROR_LOG_LOGIN_FAILED="Cookie login failed for user %
 PLG_AUTHENTICATION_COOKIE_FIELD_COOKIE_LIFETIME_LABEL="Cookie Lifetime (days)"
 PLG_AUTHENTICATION_COOKIE_FIELD_KEY_LENGTH_LABEL="Key Length"
 PLG_AUTHENTICATION_COOKIE_PRIVACY_CAPABILITY_COOKIE="In conjunction with a plugin which supports a \"Remember Me\" feature, such as the \"System - Remember Me\" plugin, this plugin creates a cookie on the user's client if a \"Remember Me\" checkbox is selected when logging into the website. This cookie can be identified with the prefix `joomla_remember_me` and is used to automatically log users into the website when they visit and are not already logged in."
-PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="Handles Joomla's cookie User authentication.<br><strong> Warning! You must have at least one other authentication plugin enabled.</strong> <br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login."
+PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="<p>Handles Joomla's cookie User authentication.</p><p><strong> Warning! You must have at least one other authentication plugin enabled.</strong><br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login.</p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_cookie.ini]:
```suggestion
PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="<p>Handles Joomla's cookie User authentication.</p><p><strong>Warning! You must have at least one other authentication plugin enabled.</strong><br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login.</p>"
```

File: administrator/language/en-GB/plg_authentication_cookie.sys.ini [hunk 4]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_AUTHENTICATION_COOKIE="Authentication - Cookie"
-PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="Handles Joomla's cookie User authentication.<br><strong> Warning! You must have at least one other authentication plugin enabled.</strong><br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login."
+PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="<p>Handles Joomla's cookie User authentication.</p><p><strong> Warning! You must have at least one other authentication plugin enabled.</strong><br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login.</p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_cookie.sys.ini]:
```suggestion
PLG_AUTHENTICATION_COOKIE_XML_DESCRIPTION="<p>Handles Joomla's cookie User authentication.</p><p><strong>Warning! You must have at least one other authentication plugin enabled.</strong><br>You will also need a plugin such as the System - Remember Me plugin to implement cookie login.</p>"
```

File: administrator/language/en-GB/plg_authentication_joomla.ini [hunk 5]
```diff
@@ -5,4 +5,4 @@
 
 PLG_AUTHENTICATION_JOOMLA="Authentication - Joomla"
 PLG_AUTHENTICATION_JOOMLA_ERR_SECRET_CODE_WITHOUT_TFA="You need to enable two factor authentication in your user profile to use the secret code field."
-PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="Handles Joomla's default User authentication.<br><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong>"
+PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="<p>Handles Joomla's default User authentication.</p><p><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_joomla.ini]:
```suggestion
PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="<p>Handles Joomla's default User authentication.</p><p><strong>Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```

File: administrator/language/en-GB/plg_authentication_joomla.sys.ini [hunk 6]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_AUTHENTICATION_JOOMLA="Authentication - Joomla"
-PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="Handles Joomla's default User authentication.<br><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong>"
+PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="<p>Handles Joomla's default User authentication.</p><p><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_joomla.sys.ini]:
```suggestion
PLG_AUTHENTICATION_JOOMLA_XML_DESCRIPTION="<p>Handles Joomla's default User authentication.</p><p><strong>Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```

File: administrator/language/en-GB/plg_authentication_ldap.ini [hunk 7]
```diff
@@ -33,4 +33,4 @@ PLG_LDAP_FIELD_V3_DESC="Default is LDAP2, but the latest versions of OpenLdap re
 PLG_LDAP_FIELD_V3_LABEL="LDAP V3"
 PLG_LDAP_FIELD_VALUE_BINDSEARCH="Bind and Search"
 PLG_LDAP_FIELD_VALUE_BINDUSER="Bind Directly as User"
-PLG_LDAP_XML_DESCRIPTION="Handles User Authentication against an LDAP server.<br><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong>"
+PLG_LDAP_XML_DESCRIPTION="<p>Handles User Authentication against an LDAP server.</p><p><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_ldap.ini]:
```suggestion
PLG_LDAP_XML_DESCRIPTION="<p>Handles User Authentication against an LDAP server.</p><p><strong>Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```

File: administrator/language/en-GB/plg_authentication_ldap.sys.ini [hunk 8]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_AUTHENTICATION_LDAP="Authentication - LDAP"
-PLG_LDAP_XML_DESCRIPTION="Handles User Authentication against an LDAP server.<br><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong>"
+PLG_LDAP_XML_DESCRIPTION="<p>Handles User Authentication against an LDAP server.</p><p><strong> Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_authentication_ldap.sys.ini]:
```suggestion
PLG_LDAP_XML_DESCRIPTION="<p>Handles User Authentication against an LDAP server.</p><p><strong>Warning! You must have at least one authentication plugin enabled or you will lose all access to your site.</strong></p>"
```

**Comment by pranotiTechjoomla** [comment]
I have tested this item :white_check_mark: successfully on 123c3bc201d2c5ae2aced8de32ca2df18b872f0a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by MeghaBiranje** [comment]
I have tested this ✅ successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by MeghaBiranje** [comment]
I have tested this item :white_check_mark: successfully on 123c3bc201d2c5ae2aced8de32ca2df18b872f0a<br /><br />I have tested this ✅ successfully <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by richard67** [comment]
@particthistle Could you apply @Quy 's suggestions (see his review comments)? Thanks in advance.

**Comment by particthistle** [comment]
@richard67 @Quy - Suggestions applied.

Please removed label:Updates Requested

Ready for testing again.

File: administrator/language/en-GB/plg_content_fields.ini [hunk 9]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_CONTENT_FIELDS="Content - Fields"
-PLG_CONTENT_FIELDS_XML_DESCRIPTION="This plugin allows you to display a custom field which has been inserted with the 'Button - Fields' plugin or using Syntax: {field #} directly into the editor area.<br><strong>Possible Syntax:</strong><br><ul><li><code>{field 1}</code> will display the field with the ID 1</li><li><code>{field 1,foo}</code> will display the selected field using the alternative layout 'foo'.</li><li><code>{fieldgroup 2}</code> will display all fields within the fieldgroup with the ID 2.</li></ul>"
+PLG_CONTENT_FIELDS_XML_DESCRIPTION="<p>This plugin allows you to display a custom field which has been inserted with the 'Button - Fields' plugin or using Syntax: {field #} directly into the editor area.</p><p><strong>Possible Syntax:</strong><br><ul><li><code>{field 1}</code> will display the field with the ID 1</li><li><code>{field 1,foo}</code> will display the selected field using the alternative layout 'foo'.</li><li><code>{fieldgroup 2}</code> will display all fields within the fieldgroup with the ID 2.</li></ul></p>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_content_fields.ini]:
```suggestion
PLG_CONTENT_FIELDS_XML_DESCRIPTION="<p>This plugin allows you to display a custom field which has been inserted with the 'Button - Fields' plugin or using Syntax: {field #} directly into the editor area.</p><p><strong>Possible Syntax:</strong></p><ul><li><code>{field 1}</code> will display the field with the ID 1</li><li><code>{field 1,foo}</code> will display the selected field using the alternative layout 'foo'.</li><li><code>{fieldgroup 2}</code> will display all fields within the fieldgroup with the ID 2.</li></ul>"
```

**Comment by richard67** [comment]
I've allowed myself to solve the conflict.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 87a8df84e6e2d7b7b70373313c059e11bc5b86f3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 87a8df84e6e2d7b7b70373313c059e11bc5b86f3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31285">issues.joomla.org/tracker/joomla-cms/31285</a>.</sub>

**Comment by Quy** [comment]
Thanks!

**Comment by particthistle** [comment]
Thanks @richard67 for fixing the final conflict and @Quy for testing again - Apologies for going dark on the last parts had my github notifications being diverted.

**Pull request merged**